README.md

```diff
@@ -1,8 +1,66 @@
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Vincent Phung . Jarvis Consulting
 
+Hello, I'm Vincent Phung, I'm currently a junior software engineer at Jarvis Consulting Group. I recently graduated with a Bachelor's degree in Computer Engineering, specializing in Software Engineering, from Toronto Metropolitan University in June 2023. I have previous experience working as a software developer intern with the Ontario Ministry of Health and Long-term Care. My main interests in the field of technology are fintech, blockchain technology, and game development. What excites me about this field is the opportunity to transform innovative ideas into tangible, impactful solutions. I find satisfaction in continuous learning, embracing new technologies, and translating them into real-world applications. In my spare time one of the things I enjoy doing is turning some of my ideas into fun projects that include full-stack web applications, scripts to automate tasks, and bots.
+
+## Skills
+
+**Proficient:** Javascript, Python, Java, Linux/Bash, HTML5/CSS3, Agile/Scrum, Git
+
+**Competent:** ReactJS, Typescript, MongoDB/NoSQL, Node.js, Express.js, Tailwind, RDBMS/MySQL/Postgres/SQL, RESTful APIs, Docker
+
+**Familiar:** C, Oracle, MATLAB, RabbitMQ, Postman
+
+## Jarvis Projects
+
+Project source code: [https://github.com/VincentP05](https://github.com/VincentP05)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/VincentP05/tree/master/linux_sql)]: A Linux cluster monitoring agent that allows the user to monitor nodes/machines and get their information if they are connected in a Linux cluster through the use of bash scripts. The information is persisted in a postgres database that's hosted on a docker container. The technologies used include Bash, Postgres, and Docker.
+
+**Core Java Apps** [[GitHub](https://github.com/VincentP05/tree/master/core_java)]:
+      
+  - Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+  - Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam.
+
+**Springboot App** [[GitHub](https://github.com/VincentP05/tree/master/springboot)]: Not Started
+
+**Python Data Analytics** [[GitHub](https://github.com/VincentP05/tree/master/python_data_anlytics)]: Not Started
+
+**Hadoop** [[GitHub](https://github.com/VincentP05/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/VincentP05/tree/master/spark)]: Not Started
+
+**Cloud/DevOps** [[GitHub](https://github.com/VincentP05/tree/master/cloud_devops)]: Not Started
```

Comment:
Please add your Grep project and comment out any not started projects or filler text.

Reply:
I have made the changes, let me know if anything else needs to be changed